src/routes/app/all-transactions.svelte

```diff
@@ -1,15 +1,20 @@
 <script lang="typescript">
 
+  import { address } from "../../stores/keyfileStore.js";
   import NavBar from "../../components/NavBar.svelte";
   import Footer from "../../components/Footer.svelte";
+  import Loading from "../../components/Loading.svelte";
   import moment from "moment";
   import { loggedIn } from "../../stores/keyfileStore.js";
   import { goto } from "@sapper/app";
   import { fade } from "svelte/transition";
+  import { equals } from "arql-ops";
```

Comment:
See my comment below as too why we need this

```suggestion
  import { and, equals } from "arql-ops";
```